python/test/unit/mesh/test_mixed_topology.py

```diff
@@ -50,18 +51,77 @@ def test_mixed_mesh_3d():
     assert len(entity_types[1]) == 1
     assert len(entity_types[2]) == 2
     assert len(entity_types[3]) == 3
-    assert CellType.triangle in entity_types[2]
-    assert CellType.quadrilateral in entity_types[2]
-
-
-def test_prism():
-    cells = [[0, 1, 2, 3, 4, 5]]
-    orig_index = [[0]]
-    ghost_owners = [[]]
-    boundary_vertices = []
-
-    topology = create_topology(MPI.COMM_SELF, [CellType.prism], cells, orig_index, ghost_owners, boundary_vertices)
-    assert len(topology.entity_types[2]) == 2
+    assert CellType.triangle == entity_types[2][1]
```

Comment:
What determines the order of `entity_types[d]`? Is it always quads then triangles? Or will this fail if the implementation changes? I couldn't find anything about the order in the docs

Reply:
At the moment, the facet order is determined in the constructor of `Topology`, which effectively does a sort, so quad(-4) appears before triangle(3). This could be changed, and the test would then fail, yes. 
The cell order, however, is arbitrary, and depends on the user input. 
So, what is the best solution? Maybe I should change the test, so it doesn't assume anything about the order?

Reply:
I think it would probably be better to avoid making the assumption in the test